docs/logging/seq-integration.md

```diff
@@ -1,109 +1,159 @@
 
-To get started with the .NET Aspire Seq integration, install the [📦 Aspire.Seq](https://www.nuget.org/packages/Aspire.Seq) NuGet package in the client-consuming project, i.e., the project for the application that uses the Seq client.
+During development, .NET Aspire runs and connects to the [`datalust/seq` container image](https://hub.docker.com/r/datalust/seq).
+
+## Hosting integration
+
+The Seq hosting integration models the server as the <xref:Aspire.Hosting.ApplicationModel.SeqResource> type. To access this type and the API, add the [📦 Aspire.Hosting.Seq](https://www.nuget.org/packages/Aspire.Hosting.Seq) NuGet package in the [app host](xref:dotnet/aspire/app-host) project.
 
 ### [.NET CLI](#tab/dotnet-cli)
 
 ```dotnetcli
-dotnet add package Aspire.Seq
+dotnet add package Aspire.Hosting.Seq
 ```
 
 ### [PackageReference](#tab/package-reference)
 
 ```xml
-<PackageReference Include="Aspire.Seq"
+<PackageReference Include="Aspire.Hosting.Seq"
                   Version="*" />
 ```
 
 ---
 
 For more information, see [dotnet add package](/dotnet/core/tools/dotnet-add-package) or [Manage package dependencies in .NET applications](/dotnet/core/tools/dependencies).
 
-## Example usage
+### Add a Seq resource
 
-<!-- TODO: <xref:Microsoft.Extensions.Hosting.AspireSeqExtensions.AddSeqEndpoint%2A> -->
-
-In the _:::no-loc text="Program.cs":::_ file of your projects, call the `AddSeqEndpoint` extension method to register OpenTelemetry Protocol exporters to send logs and traces to Seq and the .NET Aspire Dashboard. The method takes a connection name parameter.
+In your app host project, call <xref:Aspire.Hosting.SeqBuilderExtensions.AddSeq*> to add and return a Seq resource builder.
 
 ```csharp
-builder.AddSeqEndpoint("seq");
+var builder = DistributedApplication.CreateBuilder(args);
+
+var seq = builder.AddSeq("seq")
+                 .ExcludeFromManifest()
+                 .WithLifetime(ContainerLifetime.Persistent);
+
+var myService = builder.AddProject<Projects.ExampleProject>()
+                       .WithReference(seq)
+                       .WaitFor(seq);
+
+// After adding all resources, run the app...
 ```
 
-## App host usage
+> [!NOTE]
+> The Seq container may be slow to start, so it's best to use a _persistent_ lifetime to avoid unnecessary restarts. For more information, see [Container resource lifetime](../fundamentals/app-host-overview.md#container-resource-lifetime).
 
-To model the Seq resource in the app host, install the [📦 Aspire.Hosting.Seq](https://www.nuget.org/packages/Aspire.Hosting.Seq) NuGet package in the [app host](xref:dotnet/aspire/app-host) project.
+> [!IMPORTANT]
+> You must accept the [Seq End User Licence Agreement](https://datalust.co/doc/eula-current.pdf) for Seq to start.
```

Comment:
Can this be done with an environment variable in the container? Like chain a call to `.WithEnvironment("ACCEPT_EULA", true)` or something like that?